server/models/Group.js

```diff
@@ -382,8 +382,12 @@ module.exports = function(Sequelize, DataTypes) {
       },
 
       getRelatedGroups(limit) {
```

Comment:
yes, there's a better way to organize relatedGroups and supercollective. One way is to ignore the isSupercollective column and just use group.setting.superCollectiveTag. If that's set, we fetch related groups.